package.json

```diff
@@ -1,955 +1,956 @@
+                                            "type": "string",
+                                            "description": "Specify a label value to restrict this settings group to only vertices with that label."
+                                        },
+                                        "displayProperty": {
+                                            "type": "array",
+                                            "description": "The vertex property to display for the text. The first one found to have a non-empty value on the vertex is used. The default is the vertex id.",
+                                            "items": {
+                                                "type": "string"
+                                            }
+                                        },
+                                        "color": {
+                                            "type": "string",
+                                            "description": "The vertex color, either 'auto', a CSS color name or a CSS RGB value (e.g., '#0088FF')",
+                                            "pattern": "^(auto)|([a-zA-Z]+)|(#[0-9a-fA-F]{3})|(#[0-9a-fA-F]{6})$",
+                                            "default": "auto"
+                                        },
+                                        "showLabel": {
+                                            "type": "boolean",
+                                            "description": "If true, the label will be displayed along with the specified display property",
+                                            "default": true
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "default": [
+                        {
+                            "vertexSettings": [
+                                {
+                                    "displayProperty": [
+                                        "name"
+                                    ],
+                                    "color": "auto",
+                                    "showLabel": true
+                                },
+                                {
+                                    "appliesToLabel": "person",
+                                    "displayProperty": [
+                                        "fullName",
+                                        "lastName",
+                                        "firstName",
+                                        "name"
+                                    ]
+                                }
+                            ]
+                        }
+                    ]
+                }
+            }
+        }
+    },
+    "scripts": {
+        "vscode:prepublish": "npm run webpack-prod",
+        "build": "tsc -p ./",
+        "compile": "tsc -watch -p ./",
+        "pa
```

Comment:
Why did this get added as a dependency?  It always existed in the `extension.bundle.ts`

Reply:
Because one of the rules that got turned on was "no-implicit-dependencies" https://palantir.github.io/tslint/rules/no-implicit-dependencies/